src/Systmm.f90

```diff
@@ -236,6 +238,7 @@ SUBROUTINE SYSTMM(temp_file)
         !     Establish particle tracks
         !
         call Particle_Track(nr,x_head,x_bndry)
+        [I'M STILL NOT CERTAIN HOW THIS PROGRAM WORKS, BUT YOU SHOULD BE DOING PARTICLE TRACKING IN THE RIVER SEGMENTS, ONLY - JRY]
```

Comment:
The reservoir section should not need the particle tracking.  This is a "left over" part of the code from when i first modularized.  I'll put the "particle_tracking" in the "river" part of the code.